index.html

```diff
@@ -86,7 +86,8 @@ <h1 class="brand-heading">Planktos</h1>
 
     <!-- About Section -->
     <section id="about" class="container content-section">
-        <p class="text-center">Planktos enables websites to serve their static content over bittorrent from the users of the website</p>
+        <h2 class="text-center">What is Planktos?</h2>
+        <p class="text-center">Planktos enables websites to serve their static content over BitTorrent by turning users into seeders. This allows website owners to significantly reduce hosting costs for static content, scale in real-time without provisioning more web servers, and prevent user impact during an outage.<p>
```

Comment:
Pedantic, but maybe change prevent to reduce. Since we don't prevent it for a big chunk of users.